style.css

```diff
@@ -609,3 +612,102 @@ hr {
   width: 340px;
   height: 0.2px;
 }
+
+.contact-me {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 70px;
+  padding-right: 10px;
+  border-top-left-radius: 60px;
+  background-color: #6070ff;
+  width: 330px;
+  margin-top: 200px;
+
+  /* margin-left: 10px; */
+  background-image: url("./Image/contact\ form\ shapes-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 230px 730px;
+  background-position-x: 119px;
+  background-position-y: 20px;
+}
+
+.contact-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.contact-text {
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  text-align: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: -27px;
+  margin-bottom: 20px;
+}
+
+.form-container {
+  margin-left: 25px;
+}
+
+.first-name {
+  color: #172b4d;
+  padding: 10px 16px;
+  width: 260px;
+  border-radius: 8px;
+  border-width: 0;
+  margin-top: 20px;
+}
+
+.contact-textarea {
+  color: #b3bac5;
+  margin-top: 18px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 10px 16px;
+  resize: none;
+}
+
+.git-in-touch {
+  color: #6070ff;
+  margin-top: 20px;
+  margin-bottom: 45px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 8px 12px;
+  font-size: 17px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+}
+
+.git-in-touch:hover {
+  background-color: #2230d2;
+  color: #fff;
+  margin-top: 20px;
+  margin-bottom: 45px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 8px 12px;
+  font-size: 17px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+}
+
+.git-in-touch :active {
+  background-color: #070c44;
+  color: #fff;
+  margin-top: 20px;
+  margin-bottom: 45px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 12px 16px;
+  font-size: 17px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
[OPTIONAL] For CSS best practices, no need to define the `git-in-touch` class and add stylings to it. Since you already have the `normal-button` class and all the button interactions, kindly consider adding the class to the button in the HTML file and it will inherit all the features of the button. This will help make you code DRY and simple. :wink: